src/main/java/de/dennisguse/opentracks/content/data/Layout.java

```diff
@@ -1,16 +1,61 @@
 package de.dennisguse.opentracks.content.data;
 
+import android.content.res.Resources;
+import android.os.Parcel;
+import android.os.Parcelable;
+
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Collectors;
+
+import de.dennisguse.opentracks.R;
+import de.dennisguse.opentracks.settings.PreferencesUtils;
+import de.dennisguse.opentracks.util.CsvConstants;
 
-public class Layout {
+public class Layout implements Parcelable {
     private final String profile;
```

Comment:
What is the profile? It is the name for the layout?
If yes, then I would rather use name.

Reply:
Yes, it's already fixed. You were right, it was a messy. Every reference to "profile" in the code has been modified by "name" or "layoutName".